### ISSUE TITLE
The control is hidden in charts

### ISSUE THREAD

**Issue by KacperMadej** [larsac07/Timeline-Highcharts-Plugin#21]
Visible in demos, e.g. here:
http://jsfiddle.net/gh/get/library/pure/larsac07/Motion-Highcharts-Plugin/tree/master/demos/bars-demo/

Default overflow option for the main chart container is `hidden`, so it could be changed by CSS with `!import` (like in [this demo](http://jsfiddle.net/BlackLabel/6ea7osfw/)).

**Comment by Philzen** [comment]
Indeed, all the demos currently behave as if it's not doing anything unless the aforementioned CSS fix is applied. 

Which is sad, as it's an awesome plugin that many people might miss unless they realize it's a bug and how to work around it.
